src/main/kotlin/wordle/domain/Words.kt

```diff
@@ -0,0 +1,25 @@
+package wordle.domain
+
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.time.LocalDate
+import java.time.temporal.ChronoUnit
+
+const val WORD_SIZE = 5
+
+class Words {
+
+    companion object {
+        private val VALUE: List<String> = Files.readAllLines(Paths.get("src/main/resources/words.txt"))
```

Comment:
불필요한 코드 및 import문을 제거할 수 있겠네요 😄 
```suggestion
        private val VALUE: List<String> = File("src/main/resources/words.txt").readLines()
```